src/strategies/index.ts

```diff
@@ -268,6 +270,8 @@ const strategies = {
   masterchef,
   sushiswap,
   uniswap,
+  shinobi,
+  'shinobi-staked-in-tge-farms': shinobiInTGEFarms,
```

Comment:
Would be great if you can do separate PRs for two strategies. will be easy to test/review/integrate

Reply:
i intentionally did it as two separate, clean commits which each contain file changes for only a single one of the strategies, but i can separete this into two PRs if youd prefer just let me know

steps on creating strategies weren't clear on this

Reply:
alright, ive reduced it back to a single strategy

sorry for the confusion